torch/fx/graph.py

```diff
@@ -735,6 +735,22 @@ def format_arg(arg) -> str:
             else:
                 return str(arg)
 
+        def pretty_print_target(target):
+            """
+            Make target printouts more user-friendly.
+            1) builtins will be printed as `builtins.xyz`
+            2) operators will be printed as `operator.xyz`
+            3) other callables will be printed with qualfied name, e.g. torch.add
+            """
+            if isinstance(target, str):
+                return target
+            if hasattr(target, '__module__'):
+                if target.__module__ == 'builtins':
```

Comment:
I can see some corner case where a `__name__` doesn't exist here but not sure if it happens for any of these.